source/includes/databases-collections/databases-collections.php

```diff
@@ -0,0 +1,93 @@
+<?php
+require 'vendor/autoload.php';
+
+use MongoDB\BSON\Document;
+use MongoDB\Driver\ReadConcern;
+use MongoDB\Driver\ReadPreference;
+use MongoDB\Driver\WriteConcern;
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
+$client = new MongoDB\Client($uri);
+
+// Accesses the "test_database" database
+// start-access-database
+$db = $client->selectDatabase('test_database');
+// end-access-database
+
+// Accesses the "test_collection" collection
+// start-access-collection
+$collection = $client->test_database->selectCollection('test_collection');
+// end-access-collection
+
+// Explicitly creates the "example_collection" collection
+// start-create-collection
+$result = $client->test_database->createCollection('example_collection');
+// end-create-collection
+
+// Lists the collections in the "test_database" database
+// start-find-collections
+foreach ($client->test_database->listCollections() as $collectionInfo) {
+    print_r($collectionInfo) . PHP_EOL;
+}
+// end-find-collections
+
+// Deletes the "test_collection" collection
+// start-drop-collection
+$client->test_database->dropCollection('test_collection');
+// end-drop-collection
+
+// Sets read and write settings for the "test_database" database
+// start-database-settings
+$readPreference = new ReadPreference(ReadPreference::RP_SECONDARY);
+$readConcern = new ReadConcern(ReadConcern::LOCAL);
+$writeConcern = new WriteConcern(WriteConcern::MAJORITY);
+
+$db = $client->selectDatabase('test_database', [
+    'readPreference' => $readPreference,
+    'readConcern' => $readConcern,
+    'writeConcern' => $writeConcern,
+]);
+// end-database-settings
+
+// Sets read and write settings for the "test_collection" collection
+// start-collection-settings
+$readPreference = new ReadPreference(ReadPreference::RP_PRIMARY);
+$readConcern = new ReadConcern(ReadConcern::AVAILABLE);
+$writeConcern = new WriteConcern(WriteConcern::MAJORITY);
+
+$collection = $client->selectCollection('test_database', 'test_collection', [
+    'readPreference' => $readPreference,
+    'readConcern' => $readConcern,
+    'writeConcern' => $writeConcern,
+]);
+
+// end-collection-settings
+
+// Instructs the library to prefer New York data center reads using a tag set
```

Comment:
In a previous PR you mentioned that these comments are mainly used for influence AI analysis of the code example that follows. If so, should this description be more accurate?

You're actually selecting a secondary server in either NY or SF. The detail about it being a secondary and falling back to SF are what's missing.